media/static-bundles.json

```diff
@@ -432,12 +432,6 @@
       ],
       "name": "etc-firefox-retention-thank-you-referral"
     },
-    {
```

Comment:
Note for reviewer: these are all bundles where the respecitve files have already been removed in the repo. The latest gulp will error on any files not found.